src/Provider/Provider.context.ts

```diff
@@ -5,4 +5,5 @@ import { ProviderContextValue } from './Provider.interface'
 export const ProviderContext = React.createContext<ProviderContextValue>({
```

Comment:
Is it still relevant to define a context that stores labels?

Reply:
Yes I think so, otherwise we would have to add label in every `confirm` function call if we switch to another language